src/Miunie.Configuration/BotConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+using Miunie.Discord.Configuration;
+
+namespace Miunie.Configuration
+{
+    public class BotConfiguration : IBotConfiguration
+    {
+        private IConfiguration _config;
+
+        private const string DiscordBotTokenKey = "DiscordToken";
+
+        public BotConfiguration(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public string GetBotToken()
+        {
+            return _config.GetValueFor(DiscordBotTokenKey);
```

Comment:
Possibly add Lambda Expressions for simple return methods like this.
```suggestion
public string GetBotToken()
      => _config.GetValueFor(DiscordBotTokenKey);
```

Reply:
I love it, thanks! :+1: